torch/csrc/distributed/rpc/types.cpp

```diff
@@ -4,6 +4,9 @@ namespace torch {
 namespace distributed {
 namespace rpc {
 
+// It is declared in torch/csrc/distributed/rpc/types.h
+thread_local bool allowJitRRefPickle = false;
```

Comment:
Make this static and only define it in the cpp file. Add a separate method in header file `allowJitRRefPickle()` and use that elsewhere in the code.